patch/series

```diff
@@ -132,6 +132,7 @@ driver-ixgbe-external-phy.patch
 fix_ismt_alignment_issue.patch
 net-psample-module-unload.patch
 config-sensor-mp2975.patch
+Support-aaa-for-mgmt-vrf.patch
```

Comment:
Where is the patch file? Did you forget to add it with `git add`?

    stg import: [Errno 2] No such file or directory: '/__w/3/s/patch/Support-aaa-
                for-mgmt-vrf.patch'